run_freebsd.go

```diff
@@ -304,9 +304,8 @@ func setupSpecialMountSpecChanges(spec *spec.Spec, shmSize string) ([]specs.Moun
 	return spec.Mounts, nil
 }
 
-// If this function succeeds and returns a non-nil *lockfile.LockFile, the caller must unlock it (when??).
-func (b *Builder) getCacheMount(tokens []string, stageMountPoints map[string]internal.StageMountDetails, idMaps IDMaps, workDir string) (*spec.Mount, *lockfile.LockFile, error) {
-	return nil, nil, errors.New("cache mounts not supported on freebsd")
+func (b *Builder) getCacheMount(tokens []string, stageMountPoints map[string]internal.StageMountDetails, idMaps IDMaps, workDir, tmpDir string) (*spec.Mount, string, *lockfile.LockFile, error) {
```

Comment:
Nit, looks like some comments didn't get captured here as in: https://github.com/containers/buildah/pull/5965/commits/8e0bc8272b5fd221e4a9623f93e3abf84ea41933#diff-1ff945f9566b4296b404ee43b37ff16d62f13c90fe1a1dc040693524be03ea9bR968

Reply:
Good catch, I'll look into it.

Reply:
Change will be in my next push.